src/BackwardsCompatiblePayloadSerializerFactory.php

```diff
@@ -360,4 +375,24 @@ private static function addLabelName(
 
         return $serializedObject;
     }
+
+    /**
+     * @param array $serializedObject
+     * @param ReadRepositoryInterface $labelRepository
+     * @return array
+     */
+    private static function fixOrganizerLabelEvent(
+        array $serializedObject,
+        ReadRepositoryInterface $labelRepository
+    ) {
+        if (!isset($serializedObject['payload']['name'])) {
+            $uuid = $serializedObject['payload']['labelId'];
+            $label = $labelRepository->getByUuid(new UUID($uuid));
+
+            $serializedObject['payload']['label'] = $label->getName()->toNative();
```

Comment:
should this be 'name' instead of 'label', or is the isset 4 lines above using the wrong array key?